src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form-field form-field__big-bar"
+      data-qa="big"
+      action="http://localhost:8080"
+      method="post"
+    >
+      <label class="form-field__lable form-field__lable-big">
+        <input
+          data-qa="keypress"
+          class="form-field__typing form-field__typing-big"
+          type="text"
+          name="searched-big"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form
+      class="form-field form-field__small-bar"
+      data-qa="small"
+      action="http://localhost:8080"
+      method="post"
+    >
+      <label class="form-field__lable form-field__lable-small">
+
+        <input
+            class="form-field__typing form-field__typing-small"
+            type="text"
+            name="searched -small"
+            placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
```

Comment:
i don't see the difference in suggested change. maybe cause I'm reading it from the phone, but I don't see difference

Reply:
is it that crucial in writing code? just want to know for the future. cause I thought when stuff separated like that its actually easier to read

Reply:
My comment above was even more about indentation)

this (2 spaces):
```
<input
  class="example"
  type="text"
```
instead of this (4 spaces):
```
<input
    class="example"
    type="text"
```
but about the blank line - you should add it between sibling elements but not between parent and child tags for better readability. As Andriy said - it's a widespread rule that common for developers and often used in linter config

Reply:
thanks. understood